apps/meteor/client/views/admin/info/InformationPage.tsx

```diff
@@ -97,7 +98,7 @@ const InformationPage = memo(function InformationPage({
 							</Grid.Item>
 							{!showSeatCap && (
 								<Grid.Item xl={12} height='50%'>
-									<SeatsCard seatsCap={seatsCap} />
+									<SeatsCard plan={PlanName.STARTER} />
```

Comment:
should it be starter hardcoded or is it's starter by default?

Reply:
Honestly, does not matter, this is going to be removed in other PR